src/DateTimePicker/Parser.elm

```diff
@@ -28,12 +28,18 @@ parseDateTime input =
 
 skipOptionalSpaces : Parser ()
 skipOptionalSpaces =
-    Parser.ignore Parser.zeroOrMore (\c -> c == ' ')
+    Parser.chompWhile (\c -> c == ' ')
+        |> Parser.getChompedString
+        |> Parser.map (\_ -> ())
```

Comment:
🎨 If you don't care about the chomped string, you don't need to use `getChompedString`! `Parser.chompWhile (\c -> c == ' ')` will do you fine :)

Reply:
(It's not urgent at all though)